docs/_docs/About us/Executive Board's Selection Process.md

```diff
@@ -0,0 +1,57 @@
+
+### 2. Track record
+
+This is a leadership role where the exec member will have to make a lot of important decisions from strategic to tactical ones. The person will coordinate a lot with the team making sure our courses are being organized and team members are having a good time volunteering with us. 
+
+Hence, any previous leadership experience within the LDSA would be very useful and highly resemble the nature of work. This ranges from leading any particular AOR or course. There can also be certain initiatives within the organization, such as Data Science for Social Good. 
+
+### 3. Time at the organization
+
+To have a path to success as an executive member, a certain level of institutional knowledge is required. Also, we don’t want to have someone completely new joining the board immediately from the start. It’s important that we work with the person before and get to know them personally while allowing them to gain institutional knowledge of the organization as time goes by. 
+
+That being said, this is a “good-enough” criteria. So once the person has been in the organization for enough time, e.g. one year, that shouldn’t make them a lesser fit than somebody who’s been with us a year earlier. 
+
+
+## Selection Process
+
+This is our current proposal for the selection process:
+
+### 1. The team proposes potential candidates 
+A form would be shared with the team where you can elect yourself or another member. You can state and explain why you think this person or yourself would be a great fit for the position. The exec team can also propose candidates they consider would be a good fit. Note that a person proposed by the exec team will follow the exact same process as another person proposed by other team members.
+
+### 2. Candidates state interest to participate in the process
+Candidates should take into consideration the expected time availability and commitment. With respect to the candidates, the selection process should only continue once the candidate expresses their interest in continuing - this happens mostly when they are elected by others. The final number of candidates for the position should be updated with the team.
+
+### 3. Meeting with current exec board
+In this meeting, the current exec board will discuss with the candidate about the role’s responsibilities. The candidate can and should ask questions to clearly understand the role as well as the expectations. Note that at any time, the candidate can decide to withdraw from the process. 
+
+### 4. Final decision
+After the meeting and from the submitted, the executive board should have sufficient information about the candidates’ availability and why team members would like to be represented by this person. The final decision should be made together with documented reasoning on why this person was chosen and shared with everyone. 
```

Comment:
Here it is not exactly clear how this final decision is made. Does it take into consideration the number of votes from the volunteers on the candidates, for example, or is this simply a private decision from the Executive Team, on the final chosen candidates by the volunteers?